client/pages/feed.jsx

```diff
@@ -159,50 +163,56 @@ export default function Feed() {
             <div className="progress-bar">
               <div className="progress-fill"></div>
             </div>
-          </div>
+          </div> */}
           <div className="user-projects-row">
-            <img
-              className="project-ribbon-icon"
-              src="/feed-images/project-icon.svg"
-              alt="proj-icon"
-            />
-            <a className="user-projects-label">
-              <strong>Saved Projects</strong>
-            </a>
-            <div className="user-projects-count">
-              <strong>
-                {user.followingProjects ? user.followingProjects.length : "76"}
-              </strong>
-            </div>
+            <Link href="/profile">
+              <a>
+                <img
+                  className="project-ribbon-icon"
+                  src="/feed-images/project-icon.svg"
+                  alt="proj-icon"
+                />
+                <strong className="user-projects-label">My Projects</strong>
+                <div className="user-projects-count">
+                  <strong>
+                    {user.createdProjects ? user.createdProjects.length : "0"}
+                  </strong>
+                </div>
+              </a>
+            </Link>
           </div>
           <div className="user-projects-row">
-            <img
-              className="project-ribbon-icon"
-              src="/feed-images/project-icon.svg"
-              alt="proj-icon"
-            />
-            <a className="user-projects-label">
-              <strong>Updates</strong>
-            </a>
-            <div className="user-projects-count">
-              <strong>31</strong>
-            </div>
+            <Link href="/profile">
+              <a>
+                <img
+                  className="project-ribbon-icon"
+                  src="/feed-images/project-icon.svg"
+                  alt="proj-icon"
+                />
+                <strong className="user-projects-label">Saved Projects</strong>
+                <div className="user-projects-count">
+                  <strong>
+                    {user.followingProjects
+                      ? user.followingProjects.length
+                      : "0"}
+                  </strong>
+                </div>
+              </a>
+            </Link>
           </div>
-          <div className="user-projects-row">
+          {/* <div className="user-projects-row">
             <img
               className="project-ribbon-icon"
               src="/feed-images/project-icon.svg"
               alt="proj-icon"
             />
             <a className="user-projects-label">
-              <strong>My Projects</strong>
+              <strong>Updates</strong>
             </a>
             <div className="user-projects-count">
-              <strong>
-                {user.createdProjects ? user.createdProjects.length : "13"}
-              </strong>
+              <strong>{updates.length}</strong>
             </div>
```

Comment:
removed updates as they were kinda redundant (nowhere to link to since its the current page)